.github/workflows/Dockerfile.testing.yml

```diff
@@ -45,17 +46,25 @@ jobs:
     needs: build
     strategy:
       matrix:
+        # TODO when futurenet is up AND horizon released with p20/soroban support 
+        # network: [testnet, pubnet, standalone, futurenet]
+        # options: ["", "--enable-horizon-captive-core", "--enable-horizon-captive-core --enable-soroban-rpc"]
         network: [testnet, pubnet, standalone]
-        options: ["", "--enable-horizon-captive-core", "--enable-horizon-captive-core --enable-soroban-rpc"]
+        options: ["", "--enable-horizon-captive-core"]
         exclude:
           - network: testnet
             options: "--enable-horizon-captive-core"
           - network: testnet
-            options: "--enable-horizon-captive-core --enable-soroban-rpc"
+            options: "--enable-horizon-captive-core --enable-soroban-rpc" 
           - network: pubnet
             options: "--enable-horizon-captive-core"
           - network: pubnet
             options: "--enable-horizon-captive-core --enable-soroban-rpc"
+          - network: futurenet
+            options: "--enable-horizon-captive-core"
+
+
+
```

Comment:
Why do we have soroban-rpc enabled if we aren't testing it further down?

Reply:
I've left the references to it in the exclusions, so it's a no-op currently since t's not enabled in the options, this leaves one TODO line to uncomment for matrix options and network when futurenet and horizon released. otherwise, I could just have a large commented out block including the soroban-rpc/futurenet permutations of the entire matrix.

Reply:
Got it. Can you clean up the empty lines that were added here?